src/SonarScanner.MSBuild.PreProcessor/SonarWebService.cs

```diff
@@ -34,7 +34,7 @@ namespace SonarScanner.MSBuild.PreProcessor
 {
     public sealed class SonarWebService : ISonarQubeServer, IDisposable
     {
-        private const string oldDefaultProjectTestPattern = @"[^\\]*test[^\\]*$";
+        private const string OldDefaultProjectTestPattern = @"[^\\]*test[^\\]*$";
         private readonly Uri serverUri;
         private readonly IDownloader downloader;
         private readonly ILogger logger;
```

Comment:
Line 45, 188 and 221-234... what about adding extension like this, with `[CallerName]` attribute to deal with the name? It would be handy all over the project.

To clarify: Now, I'd like to create the extension and use it in this file. Not to explode
```
ArgumentNullException.ThrowIfNullOrEmpty(server)
```